frontend/ui/widget/virtualkeyboard.lua

```diff
@@ -32,14 +32,14 @@ local VirtualKey = InputContainer:new{
 }
 
 function VirtualKey:init()
-    if self.label == "Sym" or self.label == "ABC" then
-        self.callback = function () self.keyboard:setLayout(self.key or self.label) end
+    if self.label == "Sym" or self.label == "ABC" or self.label == "かな" or self.label == "記号" then
```

Comment:
Yes, that's similar to what I was thinking. Any critical notes @Hzj-jie @houqp ?